### PR TITLE
feat(widget-base): Add log config

### PR DESCRIPTION
**Pull request by bzang**
Add ability to config log verbosity

### CODE REVIEW

**Comment by bzang** [comment]
Yea. I needed to add the flag to the docs but it seemed really repetative to copy the same set of info into multiple spots so I combined them into a table. @bbender, could you give the docs a quick once over and let me know if the changes fit?

File: packages/node_modules/@ciscospark/widget-recents/README.md [hunk 1]
```diff
@@ -25,8 +24,9 @@ This widget handles coordination between your application and the Spark APIs, an
 Our widget is built using React <https://github.com/facebook/react>, Redux <https://github.com/reactjs/redux>, and the Spark Javascript SDK <https://github.com/ciscospark/spark-js-sdk>.
 
 This widget supports:
--  Listing all Spaces that the authenticated user is in.
--  Listening to incoming messages, calls, and membership events and provided those details to the developer.
+
+- Listing all Spaces that the authenticated user is in.
+- Listening to incoming messages, calls, and membership events and provided those details to the developer.
```
Comment (bbender) [inline packages/node_modules/@ciscospark/widget-recents/README.md]: Should this second bullet say this instead?

> Listening for incoming messages, calls, and membership events and providing those details to the developer.

File: packages/node_modules/@ciscospark/widget-recents/README.md [hunk 2]
```diff
@@ -66,52 +68,60 @@ For the latest builds that are pulled from the head of the master branch:
...
-1.  From the root directory run: `npm run start:package widget-recents`
+
+1. From the root directory run: `npm run start:package widget-recents`
+
+### Configuration
+
+When loading the widgets there are some configuration options you can provide:
+
+| Name | Data API | Req? | Description |
+|:--|:-------|:-:|---|---|
```
Comment (bbender) [inline packages/node_modules/@ciscospark/widget-recents/README.md]: This table doesn't render. Change this line to:
`|:--|:-------|:-:|---|`

File: packages/node_modules/@ciscospark/widget-recents/README.md [hunk 3]
```diff
@@ -66,52 +68,60 @@ For the latest builds that are pulled from the head of the master branch:
...
+| `accessToken` | `data-access-token` | Y | Access token for the user account initiating the messaging session. <br>For testing purposes you can use a developer access token from <https://developers.ciscospark.com>. |
+| `logLevel` | `data-log-level` |   | (default: `silent`) When present, widget will log debug information to console. This can be set to: `error`, `warn`, `debug`, `info`, `trace`, or `silent |
```
Comment (bbender) [inline packages/node_modules/@ciscospark/widget-recents/README.md]: The 'silent' option is missing the closing backtick.

File: packages/node_modules/@ciscospark/widget-space/README.md [hunk 4]
```diff
@@ -68,64 +71,52 @@ For the latest builds that are pulled from the head of the master branch:
...
+|:--|:-------|:-:|---|---|
+| `accessToken` | `data-access-token` | Y | Access token for the user account initiating the messaging session. <br>For testing purposes you can use a developer access token from <https://developers.ciscospark.com>. |
+| `initialActivity` | `data-initial-activity` |   | (default: `message`) Activity view to open with the widget. Available options: <ul><li>`message`: Message view</li><li>`meet`: Meet view (_Only available in 1 on 1 spaces_)</li></ul>|
+| `startCall` | `data-start-call` |   | (default: `false`) When present, widget will start in Meet view and initiate a call with the toPerson immediately. (_Only available in 1 on 1 spaces_) |
+| `logLevel` | `data-log-level` |   | (default: `silent`) When present, widget will log debug information to console. This can be set to: `error`, `warn`, `debug`, `info`, `trace`, or `silent` |
+| <td colspan=4>**Include only one of the following attributes**</td> |
+| `spaceId` | `data-space-id` |   | ID of the space you want to open. |
+| `toPersonEmail` | `data-to-person-email` |   | Email of the message recipient |
+| `toPersonId` | `data-to-person-id` |   | User Id of the message recipient |
```
Comment (bbender) [inline packages/node_modules/@ciscospark/widget-space/README.md]: This table doesn't render. Changing the header separator to this fixes it:
`|:--|:-------|:-:|---|`

Also, I can't get the `colspan` to render properly. Perhaps this should become an HTML table, or the three exclusive attributes should move into a separate table.

**Pull request merged**